pkg/resource/cluster.go

```diff
@@ -147,7 +148,10 @@ func (cluster Cluster) StatefulSetName() string {
 }
 
 func (cluster Cluster) JobName() string {
-	return fmt.Sprintf("%s-version-checker", cluster.Name())
+	return fmt.Sprintf("%s-version-checker-%d", cluster.Name(), getTimeHashInMinutes(time.Now()))
```

Comment:
check that the name is under 63 characters, otherwise shorten the name down to 63 - slug length and add the slug

Reply:
I added vchecker instead of version-checker, I open an issue for the slug